clarity-ext-scripts/clarity_ext_scripts/covid/pcr/example_result_file_rt_pcr.py

```diff
@@ -0,0 +1,89 @@
+import xlwt
+import collections
+import datetime
+import random
+from clarity_ext.extensions import GeneralExtension
+from clarity_ext_scripts.covid.parse_pcr import CT_HEADER
+
+
+class Extension(GeneralExtension):
+    def execute(self):
+        file_handle_name = "Result file"
+        timestamp = datetime.datetime.now().strftime("%y%m%dT%H%M%S")
+        user = self.context.current_user
+
+        file_name = "EXAMPLE-FILE_{}_{}.xls".format(user.initials, timestamp)
+        wb = xlwt.Workbook()
+        ws = wb.add_sheet('Results')
+
+        self.create_upper_header_info(ws)
+
+        table_row_start = 7
+        column_index = 0
+
+        # Creating the header of the table
+        for key in self.create_row():
+            ws.write(table_row_start, column_index, key)
+            column_index += 1
+
+        # Creating the content of the table
+        for well in self.context.output_container:
+            row = self.create_row(well, well.artifact)
```

Comment:
Creating row rather than table. Makes things simpler (eval is evil ;))

Reply:
True